providers-sdk/v1/plugin/service.go

```diff
@@ -24,11 +25,14 @@ type Service struct {
 
 	lastHeartbeat int64
 	heartbeatLock sync.Mutex
+
+	Cache store.KeyValue
```

Comment:
is this thread-safe? iirc w can have multiple conns using 1 provider?

Reply:
also, if we only need this for github, why not make it a github-connection specific thing?